common/src/main/java/com/wynntils/screens/guides/WynnItemGuideScreen.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.guides;
+
+import com.wynntils.screens.base.TextboxScreen;
+import com.wynntils.screens.base.WynntilsListScreen;
+import com.wynntils.screens.base.widgets.BackButton;
+import com.wynntils.screens.base.widgets.ItemSearchWidget;
+import com.wynntils.screens.base.widgets.PageSelectorButton;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.services.itemfilter.SearchQuery;
+import com.wynntils.utils.render.Texture;
+import net.minecraft.network.chat.Component;
+
+public abstract class WynnItemGuideScreen<E, B extends WynntilsButton> extends WynntilsListScreen<E, B, SearchQuery>
```

Comment:
```suggestion
public abstract class WynntilsGuideScreen<E, B extends WynntilsButton> extends WynntilsListScreen<E, B, SearchQuery>
```

Perhaps this is a better name. Alternatively, `WynntilsGuideScreenBase`.